currency.py

```diff
@@ -8,71 +8,72 @@ def EXIT(message, exit_code):
     sys.exit(exit_code)
 
 
-def date_check(date):    #we just need to check if date in range [1992-07-01;<today>] dmy
+def date_check(dateinp):
+    min_date = datetime(1992, 7, 1)
+    max_date = datetime.today()
 
-    lst[0] = int(lst[0])
-    lst[1] = int(lst[1])
-    lst[2] = int(lst[2])
-    today = datetime.today()
-
-    if date[0] > today.day and date[2] == today.year and date[1] == today.month:
-        return False
-
-    if (date[1] > today.month and date[2] == today.year) or (date[2] == 1992 and date[1] < 7):
-        return False
-
-    if date[2] < 1992 or date[2] > today.year:
-        return False
-
-    return True
+    if min_date <= dateinp <= max_date:
+        return True
+    return False
 
 
 EXIT_SUCCESS = 0
 CURRENCY_ERROR = 1
 DATE_ERROR = 2
 EXIT_REFERENCE = 3
 INPUT_ERROR = 4
+rFlag = False
 
 if len(sys.argv) == 1 or sys.argv[1] == "-h" or sys.argv[1] == "-a":
     print(
         "usage: currency.py [valute code] [date: <format{yyyy-mm-dd}>]", "\n",
         "usage example: eur 2020-01-01", "\n",
         "this script has data from 1992-07-01", "\n",
-        "other arguments: [-h] [-a] [-ec] [-exit_codes]", sep=""
+        "in default program show direct quotations", "-n"
+        "other arguments: [-h] [-a] [-e] [--exit-codes] [-r] [--reverse-quotation] [man]", sep=""
           )
-    EXIT("", EXIT_REFERENCE)
+    EXIT("help shown successful", EXIT_REFERENCE)
```

Comment:
According to PEP-8 you can't name your functions in uppercase! Try to devise a different name here